modules/kms/README.md

```diff
@@ -1,40 +1,71 @@
 # Google KMS Module
 
-Simple Cloud KMS module that allows managing a keyring, zero or more keys in the keyring, and IAM role bindings on individual keys.
+This module allows creating and managing KMS keys and IAM bindings at both the keyring and key level. An existing keyring can be used, or a new one can be created and managed by the module if needed.
```

Comment:
nit: I would change "keys" to "crypto keys" here and in other point of the module. I know that this also not straight forward in GCP documentation neither, but it would reduce confusion between crypto keys and keys version and it would be aligned with the TF resource.

Reply:
done